doc/user/content/sql/tail.md

```diff
@@ -133,21 +133,9 @@ See the [examples](#examples) for details.
 
 ### `AS OF`
 
-The `AS OF` clause specifies the time at which a `TAIL` operation begins.
+The [`AS OF`](/sql/asof) clause specifies the time at which a `TAIL` operation begins.
 See [`SNAPSHOT`](#snapshot) below for details on what this means.
```

Comment:
The fact that `AS OF` links to a separate page with the real details gets a little bit buried here. I suggest deleting this section entirely! This sentence is already in the table at the top, and the link to `AS OF` in the next section below can just go to the new standalone `AS OF` page.